src/server.c

```diff
@@ -94,19 +103,27 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
 void get_d20(int fd)
 {
     // Generate a random number between 1 and 20 inclusive
-    srand(1);
-    float d20_val = rand();
-    
-    printf("d20 roll: %f\n", d20_val);
+
     ///////////////////
     // IMPLEMENT ME! //
     ///////////////////
 
-    // Use send_response() to send it back as text/plain data
+    // time_t rawtime;
+    srand(time(NULL));
```

Comment:
nice! as a final extra: I would set this inside of main, so that you only seed once (every time the program is run) vs every single time that d20 is run.